docs/evaluation/faq/custom-evaluators.mdx

```diff
@@ -154,7 +157,39 @@ def faithfulness(run: Run, example: Example) -> dict:
     return {"key": "faithfulness", "score": average_score, "comment": comment}
 ```
 
-#### Example 4: Perplexity Evaluator
+#### Example 4: Returning Multiple Scores
+
+A single evaluator can return multiple scores. An example of when this might be useful is if you are using tool calling for an LLM-as-judge to extract multiple metrics in a single API call.
+
+```python
+from langsmith.schemas import Example, Run
```

Comment:
would stray away from langchain examples in the docs

Reply:
alright, will use the client directly